src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2621,8 +2621,12 @@ The *Signal API* is a high-level API that enables users to interact with running
 That is, it allows a job to expose specific signals (e.g., `soft_stop` to properly finish a job) that can be triggered by users from the *Workflow Execution portal* or *Scheduler portal*.
 You can try the <<_examples_2,examples>> below to better understand how it works.
 
-In summary and for simple use, just go to <<_proactive_studio, ProActive Studio>> and use the 2 predefined templates in the Controls menu: *Check_For_Signals* or *Wait_For_Signals*.
+In summary and for simple use, just go to <<_proactive_studio, ProActive Studio>> and use the predefined templates in the _Controls_ bucket, _4. Signal Templates_ project: for example: *Check_For_Signals*, *Wait_For_Signals* or *Wait_For_Signal_With_Variables*.
 Fill up the Task Variable *SIGNALS*, and use those templates in your own workflow to be able to receive input from, for instance, the Workflow Execution portal or Scheduler portal.
+*Check_For_Signals* is a template workflow that sends a ready notification for all the signals specified in the variable SIGNALS, then loops until one signal among those specified is received by the job.
+*Wait_For_Signals* also sends a ready notification for all the signals specified in the variable SIGNALS, loops until one signal among those specified is received by the job and then removes the signal.
+*Wait_For_Signal_With_Variables* is template workflow that waits until one signal (with input parameters) that is specified in the variable SIGNAL is added to the set of job signals.
```

Comment:
I suggest to add few words for each workflow template to explain its main difference compared to the others, and (possibly) in  which case to use it